examples/python/agents/multi-provider-agent.py

```
@@ -0,0 +1,285 @@
+"""
+Multi-Provider/Multi-Model Agent Example
+
+This example demonstrates how to use multiple AI providers/models with intelligent
+agent-based selection for cost optimization and performance.
+"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)